test_data/yast/encryption/home_encrypted.yaml

```diff
@@ -0,0 +1,18 @@
+---
+crypttab:
+  num_devices_encrypted: 1
+cryptsetup:
+  device_status:
+    message: is active and is in use.
+    properties:
+      type: LUKS1
+      cipher: aes-xts-plain64
+      device: /dev/vda3
+      key_location: dm-crypt
+      mode: read/write
+backup_file_info: 'LUKS encrypted file, ver 1 \[aes, xts-plain64, sha256\]'
+cr_home:
+  backup_path: '/root/bkp_luks_header_cr_home'
+partitioner_table: 'hard-disks'
+partitioner_proposal: current
+partition: vda3
```

Comment:
I fail to see where are all these test data used, can you please help?

Reply:
partitioner_table -> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11037/files/72d26f05e030c757730ce63a8e97a2fac79b1e5b#diff-9e816c002ea7dab6fb625f8bf567cfdaR214
partitioner_proposal -> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11037/files#diff-d480378d7a0760e5b1f68e4eed6f1bf4R30
partition -> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11037/files#diff-9e816c002ea7dab6fb625f8bf567cfdaR217

Reply:
Yes, but this is just three of them. What is the purpose of the rest?

Reply:
The rest are used in the `validate_encrypt`. Pretty much we use the same set of test_data in all of those sort of validations